spyder/utils/encoding.py

```diff
@@ -245,10 +245,15 @@ def write(text, filename, encoding='utf-8', mode='wb'):
             umask = os.umask(0)
             os.umask(umask)
             original_mode = 0o777 & ~umask
-        with atomic_write(filename,
-                          overwrite=True,
-                          mode=mode) as textfile:
-            textfile.write(text)
+        try:
+            with atomic_write(filename, overwrite=True,
+                              mode=mode) as textfile:
+                textfile.write(text)
+        except OSError:
+            # Some filesystems don't support the option to sync directories
+            # issue untitaker/python-atomicwrites#17
+            with open(filename, mode) as textfile:

```

Comment:
Let's change this to be

```python
if error.errno != errno.EINVAL:
    with open(filename, mode) as textfile:
        textfile.write(text)
```

and import the `errno` module at the top of this file (it's a standard library module).

The idea is to capture the specific error generated when syncing directories fail and let the others pass.